Uplift/Views/GymDetail/FitnessCenter/GymDetailFitnessCenterHoursCell.swift

```diff
@@ -2,13 +2,238 @@
 //  GymDetailFitnessCenterHoursCell.swift
 //  Uplift
 //
-//  Created by alden lamp on 10/22/23.
+//  Created by alden lamp on 10/23/23.
 //  Copyright © 2023 Cornell AppDev. All rights reserved.
 //
 
 import Foundation
 import UIKit
+import SnapKit
+
+protocol GymDetailFitnessCenterHoursCellDelegate: AnyObject {
+    func didDropHours(isDropped: Bool, completion: @escaping () -> Void)
+}
 
 class GymDetailFitnessCenterHoursCell: UITableViewCell {
+
     static let reuseId = "GymDetailFitnessCenterHoursCellReuseId"
+
+    enum LayoutConstants {
+        static let hoursTableViewRowHeight: CGFloat = 27
+        static let hoursTableViewBottomRowHeight: CGFloat = 19
+        static let hoursTableViewTopPadding: CGFloat = 12
+    }
+
+    private var isDisclosed = false
+    private var hours: OpenHours!
+    private weak var delegate: GymDetailFitnessCenterHoursCellDelegate?
+
+    private var tableViewHeight: ConstraintMakerEditable?
+
+    // MARK: - Private view vars
+    private let hoursTitleLabel = {
```

Comment:
Haven't seen views initialized like this in a long time haha

Reply:
Yeah lol its my personal preference. Feels cleaner and easier to read, but I can change that if needed.